source/MRViewer/MRRibbonButtonDrawer.cpp

```diff
@@ -3,6 +3,7 @@
 #include "MRColorTheme.h"
 #include "MRRibbonConstants.h"
 #include "MRImGuiImage.h"
+#include "MRShortcutManager.h"
```

Comment:
why this?

Reply:
because of forward declaration of ShortcutManager in ImGuiMenu.h